charts/workflow-config/templates/copy-stac-asset-s3-secret.yaml

```diff
@@ -9,4 +9,7 @@ data:
   secret_access_key: {{ .Values.s3.secretAccessKey }}
   region: {{ .Values.s3.region }}
   session_token: {{ .Values.s3.sessionToken }}
+  data_bucket: {{ .Values.s3.data_bucket }}
+  api_url: {{ .Values.s3.api_url }}
```

Comment:
Do we need to remove the api_url in the secret?

Reply:
Done